spec/src/main/asciidoc/integration.adoc

```diff
@@ -19,18 +19,28 @@
 
 == SDK integration
 
-Implementations SHALL provide integration by appropriately configuring compatible OpenTelemetry SDK for application runtime.
-Regardless of signal type being considered -- traces or metrics -- following requirements SHALL be met:
+Implementations SHALL provide integration by appropriately configuring one or more compatible OpenTelemetry SDK instances for the application runtime.
+Regardless of signal type being considered -- traces, logs or metrics -- the following requirements SHALL be met.
 
 [[sec:enabling-opentelemetry-support]]
 === Enabling OpenTelemetry support
 By default, MicroProfile Telemetry is deactivated.
 
-In order to enable any of aspects of integration, the configuration `otel.sdk.disabled=false` MUST be specified in any of the configuration sources available via MicroProfile Config.
+In order to enable any aspects of integration, the configuration `otel.sdk.disabled=false` MUST be specified.
+
+If the `otel.sdk.disabled=false` configuration setting is visible to the runtime at initialization time then the runtime MUST provide a SINGLE OpenTelemetry SDK instance which MUST be used by the runtime and all applications.
+
+If the `otel.sdk.disabled=false` configuration setting is only visible to the application(s) at initialization time, then runtime telemetry MUST be disabled and the application(s) MUST be configured using the visible `otel.*` properties.
+
+Implementations that do not support use of MicroProfile Config during runtime initialization may use the `OTEL_SDK_DISABLED` environment variable or `otel.sdk.disabled` Java system property to specify this setting.
+
+At application initialization time runtimes MUST use configuration sources available via MicroProfile Config for configuration.
+
+NOTE: This is a breaking change for runtimes that can simultaneously run multiple applications. In MicroProfile Telemetry 1.1 the `OTEL_SDK_DISABLED` environment variable could be used to indicate whether to enable or disable the application instance(s) of OpenTelemetry. Users that want to enable/disable individual application instances can do so using any MicroProfile Config configuration source that is only visible to applications. Users that want to enable an OpenTelemetry instance to be used by the runtime and all applications need to provide any OpenTelemetry configuration settings in a way that is visible to the runtime at initialization time (for example, using environment variables).
```

Comment:
```suggestion
NOTE: This is a breaking change for runtimes that can simultaneously run multiple applications.
In MicroProfile Telemetry 1.1 the `OTEL_SDK_DISABLED` environment variable could be used to indicate whether to enable or disable the application instance(s) of OpenTelemetry.
Users that want to enable/disable individual application instances can do so using any MicroProfile Config configuration source that is only visible to applications.
Users that want to enable an OpenTelemetry instance to be used by the runtime and all applications need to provide any OpenTelemetry configuration settings in a way that is visible to the runtime at initialization time (for example, using environment variables).
Runtimes MAY provide additional means of configuring per-application SDKs or runtime extensions.
```

As mentioned in the issue discussion I think it is important to mention the possibility of possible variations for special cases. (And formatting to one line per sentence is just recommended practice :))

Reply:
All of the above lines get rendered into the same NOTE block by asciidoc (the line breaks don't seem to change the formatting, at least in VS Code preview). I think your last line would be better to put above the NOTE (since it's not part of the description of the breaking change).  I'll add a commit to do that.